src/Components/NavBar/Menus/__tests__/DropDownMenu.test.tsx

```diff
@@ -46,4 +47,19 @@ describe("DropDownMenu", () => {
 
     expect(dropDownSection.length).toBe(5)
   })
+
+  xit("tracks analytics click events correctly", () => {
+    const wrapper = getWrapper()
+    wrapper
+      .find(MenuItem)
+      .first()
+      .simulate("click")
+
+    expect(trackEvent).toHaveBeenCalledWith({
+      action_type: "Click",
+      context_module: "HeaderArtworksDropdown",
+      subject: "New this Week",
+      destination_path: "/collection/new-this-week",
+    })
+  })
```

Comment:
This seem pretty legit but I can't seem to get this spec to pass. The analytics event also works in the browser.

<img width="658" alt="Screen Shot 2020-04-15 at 2 14 40 PM" src="https://user-images.githubusercontent.com/5201004/79372171-9a728200-7f23-11ea-81e7-77f532a74a7a.png">